azurerm/config.go

```diff
@@ -411,6 +415,7 @@ func getArmClient(c *authentication.Config, skipProviderRegistration bool, partn
 	client.registerCosmosAccountsClients(endpoint, c.SubscriptionID, auth)
 	client.registerDatabases(endpoint, c.SubscriptionID, auth, sender)
 	client.registerDataLakeStoreClients(endpoint, c.SubscriptionID, auth)
+	client.registerFrontdoorInstanceClients(endpoint, c.SubscriptionID, auth)
```

Comment:
Could we move the registration to the internal service package like can be seen above?